src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private String TAGGING_STRING_FORMAT = "%s %s %s";
+    private Person person;
+    private Tag tag;
+    private boolean isAdd;
+    
+    public Tagging(Person person, Tag tag, boolean isAdd) {
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.